### DISCUSSION TITLE
Discussions in private repo can't create new discussion

### DISCUSSION THREAD

**Discussion by thanhcsf** [community/community#3577] [Discussions] [answered]
I can't create a new discussion in a private repository, I get the message "You can't perform that action at this time.". What should I do in this case?

<img width="1381" alt="Screen Shot 2021-03-19 at 17 02 47" src="https://user-images.githubusercontent.com/8481767/111763613-f03e6f80-88d4-11eb-9c6e-e6cb4a948a0c.png">

**Comment by PranamBhat** [comment]
Hi @duythanhvn 

Go to your Repository settings and under "Features", click on "Discussions". Save your changes and refresh your page. Now try the same.

![image](https://user-images.githubusercontent.com/32120033/111666084-62617680-8839-11eb-8e8a-f3b977806ad4.png)


Note :  GitHub officially launched **Discussions** as a beta for all public repositories to opt into. You can view more information here in our [changelog](https://github.blog/changelog/2020-12-08-github-discussions-now-available-as-a-public-beta/) post as well as the [blog post](https://github.blog/2020-12-08-new-from-universe-2020-dark-mode-github-sponsors-for-companies-and-more/).

For some additional details regarding next steps regarding private repositories and specifics around upvoting etc please reference this thread [https://github.com/github/feedback/discussions/24#discussioncomment-179134](https://github.com/github/feedback/discussions/24#discussioncomment-179134)

**Reply by thanhcsf** [reply]
Hi @PranamBhat 

Thanks for the answers, but you seem to have misunderstood my question with the Github team. I was having issues with Discussions in **the private repository**, and I opened it up in the configuration section and got the same problem as I reported above.

Thank you.

**Reply by tomaszmichalak** [reply]
Agree, it does not work for private repositories. I can see "discussions" tab but there is no "Start a new discussion" button. I checked this guide first: https://docs.github.com/en/discussions/quickstart.

**Comment by thanhcsf** [comment, marked as answer]
Now that I get back, I have created a new discussion in my private repository. I don't know why, but it worked. Thanks, everyone.

**Comment by PranamBhat** [comment]
Hope your issue has been resolved. Kindly mark this as "Answered" and leave the discussion. Thank you.

**Comment by tomaszmichalak** [comment]
@PranamBhat Thanks, works as expected.

**Reply by tomaszmichalak** [reply]
@PranamBhat where is the button to mark the thread as "Answered"?

**Reply by PranamBhat** [reply]
Only Repo owners can mark Discussions as answers as well as triagers, maintainers and the author of a post. Here, @duythanhvn seems to be the owner.

@duythanhvn  There is a right mark on top right corner of the particular comment.
